components/dev-tools/spack/SPECS/spack.spec

```diff
@@ -84,3 +89,5 @@ rm -rf $RPM_BUILD_ROOT
 %defattr(-,root,root,-)
 %{OHPC_HOME}
 %doc LICENSE README.md
+
+%changelog
```

Comment:
We just removed the %changelog section from all SPEC files. Please do not add it back.

Reply:
Oops - artifact from forking before the cleanup got pushed - fixed!